### ISSUE TITLE
OnOffState for HA Light

### ISSUE THREAD

**Issue by mlamoure** [FlyingDiver/Indigo-HA-Agent#10]
I noticed this minor issue with the HA Light.

Notice that the custom states information is correct - the light is on.  Brightness is set to 59.
However, the Indigo device state information is wrong.  It thinks the light is off.  Slider is at 0.

(Update): Also if I make a brightness change via Indigo, the command successfully sends, but then Indigo quickly reverts to Off / 0 for the state.

Note: I'm on Indigo 2022.2 as I wait for one last plugin to convert to python 3.

<img width="1151" alt="image" src="https://github.com/FlyingDiver/Indigo-HA-Agent/assets/1021890/f60bb692-fb93-4732-897c-0314de8bca19">

**Comment by mlamoure** [comment]
Here's the device info dump.

```
Jan 2, 2024 at 2:40:00 PM
   Embedded script executor host2 started
   Script                          address : light.basement_office_work_light
batteryLevel : None
blueLevel : None
brightness : 0
buttonConfiguredCount : 0
buttonGroupCount : 0
configured : True
defaultBrightness : 100
description : 
deviceTypeId : HAdimmerType
displayStateId : brightnessLevel
displayStateImageSel : DimmerOff
displayStateValRaw : 0
displayStateValUi : 0
enabled : True
energyAccumBaseTime : None
energyAccumTimeDelta : None
energyAccumTotal : None
energyCurLevel : None
errorState : 
folderId : 0
globalProps : MetaProps : (dict)
     no.homeassistant.plugin : (dict)
          SupportsStatusRequest : false (bool)
          address : light.basement_office_work_light (string)
          states_list : (list)
               Item : min_color_temp_kelvin (string)
               Item : max_color_temp_kelvin (string)
               Item : min_mireds (string)
               Item : max_mireds (string)
               Item : supported_color_modes (string)
               Item : color_mode (string)
               Item : brightness (string)
               Item : color_temp_kelvin (string)
               Item : color_temp (string)
               Item : hs_color (string)
               Item : rgb_color (string)
               Item : xy_color (string)
               Item : friendly_name (string)
               Item : supported_features (string)
greenLevel : None
id : 735116060
lastChanged : 2024-01-01 18:40:07
lastSuccessfulComm : 2024-01-01 18:40:07
ledStates : []
model : Home Assistant Light
name : Basement Office Work Light
onBrightensToDefaultToggle : False
onBrightensToLast : False
onState : False
ownerProps : no.homeassistant.plugin : (dict)
     SupportsStatusRequest : false (bool)
     address : light.basement_office_work_light (string)
     states_list : (list)
          Item : min_color_temp_kelvin (string)
          Item : max_color_temp_kelvin (string)
          Item : min_mireds (string)
          Item : max_mireds (string)
          Item : supported_color_modes (string)
          Item : color_mode (string)
          Item : brightness (string)
          Item : color_temp_kelvin (string)
          Item : color_temp (string)
          Item : hs_color (string)
          Item : rgb_color (string)
          Item : xy_color (string)
          Item : friendly_name (string)
          Item : supported_features (string)
pluginId : no.homeassistant.plugin
pluginProps : emptyDict : (dict)
protocol : Plugin
redLevel : None
remoteDisplay : True
sharedProps : com.indigodomo.indigoserver : (dict)
states : States : (dict)
     actual_state : on (string)
     brightness : 122 (integer)
     brightnessLevel : 0 (integer)
     color_mode : color_temp (string)
     color_temp : 250 (integer)
     color_temp_kelvin : 4000 (integer)
     friendly_name : Basement Office Work Light (string)
     hs_color : [26.812, 34.87] (string)
     lastUpdated : 2024-01-01T23:40:06.017218+00:00 (string)
     max_color_temp_kelvin : 6993 (integer)
     max_mireds : 344 (integer)
     min_color_temp_kelvin : 2906 (integer)
     min_mireds : 143 (integer)
     onOffState : off (on/off bool)
     rgb_color : [255, 205, 166] (string)
     supported_color_modes : ["color_temp"] (string)
     supported_features : 0 (integer)
     xy_color : [0.421, 0.364] (string)
subModel : 
subType : 
supportsAllLightsOnOff : False
supportsAllOff : False
supportsColor : False
supportsOnState : True
supportsRGB : False
supportsRGBandWhiteSimultaneously : False
supportsStatusRequest : False
supportsTwoWhiteLevels : False
supportsTwoWhiteLevelsSimultaneously : False
supportsWhite : False
supportsWhiteTemperature : False
version : None
whiteLevel : None
whiteLevel2 : None
whiteTemperature : None

```

**Comment by FlyingDiver** [comment]
Fixed in 2023.4.1.  Requires Indigo 2023.2
